api/client.js

```diff
@@ -182,6 +182,19 @@ function connectAndLogin(url, credentials, options, callback) {
 }
 
 
+/**
+  Returns a url that is to be used to connect to a supplied model uuid on the
+  supplied controller url.
+  @param {String} controllerURL The url that's used to connect to the controller.
+    The `connect` method handles redirections so the public URL is fine.
```

Comment:
This sentence seems weird without a bit of context. Also the connect method does not handle redirections, connectAndLogin does.

Reply:
updated